incubator/hnc/docs/releasing.md

```diff
@@ -133,27 +150,39 @@ warning. If this was a release candidate, you're done.
 Otherwise, update the [README](../README.md#start) and [user
 guide](user-guide/how-to.md#admin-install) to refer to your new release.
 
-Finally, if this is a minor (not patch) release, remove any obsolete
-documentation from the guide - e.g., if you've just released 0.6, you can remove
-references to 0.4. Add a link in the [front page](user-guide/README.md) to the
-guide on the branch that you've just removed.
-
-On the other hand, if this was a patch release _and you need to document
-something_, ensure you document it on _both_ the master _and_ the release
-branch.
-
-We may revise this guidance as HNC matures.
+If this was a patch release _and you need to document something_, ensure you
+document it on _both_ the master _and_ the release branch.
 
 ## Update Krew
 
 Starting with HNC v0.6.x, the build process also generates a Krew tarball and
 manifest. This manifest should be downloaded and checked into the Krew index,
-*if* it's for the latest branch. E.g. if you've already released HNC v0.7.0 and
-have to release HNC v0.6.1, do *not* update the Krew index; Krew can only
-support one version of a plugin at a time so we should only support the most
-recent branch.
-
-More details on updating Krew TBD.
+*if* it's for the latest branch (and is not a release candidate). E.g. if you've
+already released HNC v0.7.0 and have to release HNC v0.6.1, do *not* update the
+Krew index; Krew can only support one version of a plugin at a time so we should
+only support the most recent branch.
+
+Once the Krew manifest has been generated, [test it
+locally](https://krew.sigs.k8s.io/docs/developer-guide/testing-locally/):
+download the manifest and try to install it via `kubectl krew install
+--manifest=FILENAME` (do _not_ override the archive as you want to use the
+released archive). Once it's working, create a PR to
+https://github.com/kubernetes-sigs/krew-index, such as [this
+one](https://github.com/kubernetes-sigs/krew-index/pull/890).
+
+If you like, you can strike out the Krew installation instructions on the
+release notes until this PR is approved (include a link to the PR in the
+relnotes page until that's done). You don't have to do this if the _last_
```

Comment:
`relnotes` is an short term on purpose or a typo?

Reply:
It was a short form but I fixed it.